src/components/UI/organisms/UserRegistrationForm.js

```diff
@@ -1,20 +1,47 @@
 import {Form, Button, Row, Col} from 'react-bootstrap';
+import React from 'react';
+
 const UserRegistrationForm = (props) => {
```

Comment:
Looking at vega-spring, seems like registration should take `firstname`, `lastname`, `password`, and `username`, so maybe we need to change this form? not sure why we are taking a message upon sign up?